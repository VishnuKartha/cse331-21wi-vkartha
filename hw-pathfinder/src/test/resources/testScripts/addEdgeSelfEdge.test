# makes sure that you can added self edges(edges where the parent and the child are the same)

# Create a Graph
CreateGraph graph1

# Add a nodes
AddNode graph1 n1

# adds a self edge
AddEdge graph1 n1 n1 2020
