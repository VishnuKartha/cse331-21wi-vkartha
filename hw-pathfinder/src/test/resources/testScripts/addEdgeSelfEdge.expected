# makes sure that you can added self edges(edges where the parent and the child are the same)

# Create a Graph
created graph graph1

# Add a nodes
added node n1 to graph1

# adds a self edge
added edge 2020.000 from n1 to n1 in graph1